payment-handler/can-make-payment-event.https.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<!-- Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<meta charset="utf-8">
+<title>Tests for CanMakePaymentEvent</title>
+<link rel="help" href="https://w3c.github.io/payment-handler/#the-canmakepaymentevent">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+promise_test(async t => {
+  await navigator.serviceWorker.register('app-can-make-payment-false.js');
+  const registration = await navigator.serviceWorker.ready;
+  await registration.paymentManager.instruments.set('instrument-key', {
+    name: 'Test Payment Method',
+    enabledMethods: 'https://w3c-test.org',
+  });
+  await promise_rejects(t, 'NotSupportedError', (new PaymentRequest([{
+    supportedMethods: 'https://w3c-test.org',
+  }], {
+    total: {
+      label: 'Total',
+      amount: {
+        currency: 'USD',
+        value: '0',
+      },
+    },
+  })).show());
+  await registration.unregister();
+}, 'If CanMakePaymentEvent.respondWith(false) is called, then the payment method is not supported.');
+
+promise_test(async t => {
+  await navigator.serviceWorker.register('app-can-make-payment-true.js');
+  const registration = await navigator.serviceWorker.ready;
+  await registration.paymentManager.instruments.set('instrument-key', {
+    name: 'Test Payment Method',
+    enabledMethods: 'https://w3c-test.org',
+  });
+  const request = new PaymentRequest([{
+    supportedMethods: 'https://w3c-test.org',
+  }], {
+    total: {
+      label: 'Total',
+      amount: {
+        currency: 'USD',
+        value: '0',
+      },
+    },
+  });
+  const acceptPromise = request.show();
+  await request.abort();
+  await promise_rejects(t, 'AbortError', acceptPromise);
+  await registration.unregister();
+}, 'If CanMakePaymentEvent.respondWith(true) is called, then the payment method is supported.');
+</script>
```

Comment:
I was expecting to see `request.canMakePayment()` tests here. That works with `CanMakePaymentEvent`, right?

Reply:
Added the request.canMakePayment() calls as well.